src/coord/src/coord.rs

```diff
@@ -544,6 +550,28 @@ impl Coordinator {
         }
         // These can occasionally be equal, so ignore the update in that case.
         if next_ts > self.closed_up_to {
+            // Close out the timestamp for persisted tables.
+            //
+            // TODO: Allow sealing multiple streams at once to reduce the
+            // overhead.
+            let (tx, rx) = std::sync::mpsc::channel();
```

Comment:
This function is called about 5x per second. Is the below performance ok?

Reply:
Ok. Does it make sense to put this new code behind that flag then? It would prevent needless creation of a channel and iterating over the entire catalog many times per second.